arelle/UrlUtil.py

```diff
@@ -385,3 +386,10 @@ def relativeUri(baseUri: str, relativeUri: str) -> str: # return uri relative to
     if mBaseUri and not mRelUri:
         baseUri = mBaseUri.group(1) # remove the zip part so relative URI is within zip
     return os.path.relpath(relativeUri, os.path.dirname(baseUri)).replace('\\','/')
+
+
+def decodeBase64DataImage(imageData: str | None) -> bytes | None:
```

Comment:
I'll write up a ticket. Initially I was going to do that here, but decided to keep the footprint of this PR as small as possible to avoid conflicts with the other ongoing ESEF work